apps/test/unit/music/RandomSkipManagerTest.js

```diff
@@ -0,0 +1,74 @@
+/* Tests for music/player/RandomSkipManager.js */
+
+var assert = require('assert');
+import RandomSkipManager from '@cdo/apps/music/player/RandomSkipManager';
+
+describe('Music random skip manager tests', function() {
+  it('nested test', function() {
+    const randomSkipManager = new RandomSkipManager();
+
+    randomSkipManager.init();
+
+    assert.deepEqual(randomSkipManager.getSkipContext(), {
+      insideRandom: false,
+      skipSound: false
+    });
+
+    // prettier-ignore
```

Comment:
nit: mind adding a comment why this is needed? I'm assuming it's to maintain the indentation corresponding to the stack, right?

Reply:
That's right; will do.